src/pages/en/guides/data-fetching.md

```diff
@@ -38,11 +38,11 @@ const randomUser = data.results[0]
 
 Astro can also use `fetch()` to query a GraphQL server with any valid GraphQL query.
 
-```astro title="src/components/Weather.astro"
+```astro title="src/components/Weather.astro" "await fetch"
```

Comment:
The reason I stopped there is because the actual end of the GraphQL `fetch` call is waaaay further down and I didn't want to highlight the entire block as it seemed too much. 😄

My logic was: If I did highlight function arguments, I'd have to highlight them all in order not to mislead people. And because the arguments were too long I chose not to.